### PR TITLE
Mood Changes: Reduced Penalty, Increased Insanity Tolerance

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
This makes it take slightly longer to go insane
The movement reduction is reduced by 30%
The crit threshold modifier is also reduced 30%

## Why It's Good For The Game
Mood is a dumb mechanic, where I go insane from slipping on a banana peel or not eating on time, meanwhile the station is on fire and I have other shit to tend to, like playing Despacito in the hallway as the world's greatest mime, however this is not related to my sanity or lack thereof.

## Changelog
:cl:
tweak: Insanity threshold up from 25->18
tweak: Insanity penalties reduced
/:cl:

### CODE REVIEW

**Comment by Toxici11i** [comment]
+1 slow is gay

**Comment by qwertyquerty** [comment]
I don't think the speed modifier should be removed, just lessened. YA NEED THE GOOD ATTITUDE

**Comment by jlsnow301** [comment]
I don't really like the mechanic at all, however I do want to make hunger more consequential, would it be balanced if I made a slow effect in hunger rather than mood? Cooks need love too

**Comment by qwertyquerty** [comment]
I mean hunger is directly related to mood

**Comment by jlsnow301** [comment]
It's a component of. Environmental, drugs, beer, slipping, all are also

**Comment by jlsnow301** [comment]
I almost wish it was just a flavor text. Laugh out loud occasionally, or cry. But a full movement speed pen is hard.

**Comment by ike709** [comment]
👎 

Balance the penalty threshold and the intensity of various events sure, but removing the downsides to bad mood removes the point of having mood enabled in the config in the first place.

**Comment by jlsnow301** [comment]
It shouldn't be lol. How's if I changed the values and reduced the penalty?

**Comment by ike709** [comment]
Ideally there would be a code bounty for a larger rework of the mood system so that the two main influences on mood are semi-realistic needs (hunger, being relatively clean, and the like) and traumatizing events (seeing several crew burned alive in a plasma flood, for example) rather than being slowly driven insane by a dozen minor inconveniences and trying to counter it with a dozen minor mood-boosters (as the system is designed now).

Alternatively, just make the RP server happen and disable mood on the main server.

But both of those are out of the scope of this PR so I'm okay with bandaiding it by reducing the penalty and/or increasing the insanity threshold.

**Comment by jlsnow301** [comment]
Adjusted.

File: code/datums/components/mood.dm [hunk 1]
```diff
@@ -214,16 +214,16 @@
 	var/mob/living/master = parent
 	switch(sanity)
 		if(SANITY_INSANE to SANITY_CRAZY)
-			setInsanityEffect(MAJOR_INSANITY_PEN)
```
Comment (ike709) [inline code/datums/components/mood.dm:217]: Nerf the define values but don't remove the effects outright for major or minor insanity.

**Review by ike709 — APPROVED** [review]
Good for vote or merge at qwerty's discretion.

File: code/datums/components/mood.dm [hunk 2]
```diff
@@ -137,7 +137,7 @@
 				highest_absolute_mood = absmood
 
 	if(!conflicting_moodies.len) //no special icons- go to the normal icon states
-		if(sanity < 25)
+		if(sanity < 32)
```
Comment (qwertyquerty) [inline code/datums/components/mood.dm]: Why did you change this?

File: code/datums/components/mood.dm (hunk 1 above)
Comment (jlsnow301) [reply code/datums/components/mood.dm:217]: Done

File: code/datums/components/mood.dm (hunk 2 above)
Comment (ike709) [reply code/datums/components/mood.dm]: Why is this a magic number in the first place?

Comment (jlsnow301) [reply code/datums/components/mood.dm]: This is the threshold. I originally upped it to 30 because I also reduced the base modifier of ALL mood adjustments by 20%. In combination, that felt huge. 
However, the overall modifier is returned, so I felt I had more leeway

Comment (qwertyquerty) [reply code/datums/components/mood.dm]: Are you sure this is what you meant to do? Thats a < symbol not >

Comment (ike709) [reply code/datums/components/mood.dm]: Shouldn't this be `if(sanity < SANITY_INSANE)` ?

Comment (jlsnow301) [reply code/datums/components/mood.dm]: Nice catch Qwerty. Changed that.
